### DISCUSSION TITLE
Timeline with the Roadmap?

### DISCUSSION THREAD

**Discussion by boephe** [osTicket/roadmap#36] [Q&A] [answered]
Hi there,

Does your roadmap have a timeline as well? I looked at the project and there was no Done phase so it looks like none of the tasks are complete, is that correct? In April you said you were in a cooldown phase and phases were about 6 weeks long, can you give us an update where you are up to now?

We are looking at overhauling our ticket system and are deciding between updating osTicket or moving to something new. We'd really like to know if osTicket version 2 is close enough that we should wait for that or should we asses the currently available version?

Any information or update would be great.

Phoebe

**Comment by JediKev** [comment]
@boephe 

No, we do not have a timeline. While we are working hard it will take some time as this is a complete codebase rewrite.

The v2.0 repo is private currently so you are unable to see the full progress. We hope to update the roadmap soon to reflect the current status.

We are always working hard but as you may know we have a current (legacy) codebase that we still have to maintain. We just got done with a major release in legacy that took a while to complete. So now that we are done with that we are re-focusing on v2.0.

I couldn’t give you a percentage of completion at this time as there is just so many moving parts. If I gave you a percentage or timeline I’d be lying as we just don’t know at this time.

If you are going to assess osTicket you’d have to assess the current legacy codebase.

Cheers.

**Reply by boephe** [reply]
@JediKev 

Thanks for your prompt reply. I appreciate the large project, hope things are going well and looking forward to a great product 😄 

Blessings,

Boephe

**Reply by mbassiouny33** [reply]
@JediKev , out of curiosity, if I may ask, are you expecting months? or years? (panned for 2023? or possibly later) ,no rush, just curious. Thanks!

**Reply by JediKev** [reply]
@mbassiouny33 

There is still no set timeline. As I mentioned before I couldn’t give you a percentage of completion at this time as there is just so many moving parts. If I gave you a percentage or timeline I’d be lying as we just don’t know at this time. With that being said the traction is picking up as we progress further.

Once we update the roadmap it will reflect the current status and progress.

Cheers.

**Reply by mbassiouny33** [reply]
thanks for the info!

**Reply by SCT-Ri** [reply]
@JediKev Hello, now more than two years later: Can you give us an indication of the current status?

**Reply by JediKev** [reply]
@SCT-Ri 

You can read here:
- https://github.com/osTicket/roadmap/discussions/66#discussioncomment-11787604

Cheers.

**Comment by makumba8888** [comment]
looking at the last comments here, which date from  about 13 months, for those of us trying to decide if we keep using the software or go with a newer tool, the decision is getting easier to make by the month. OST was already looking old a few years ago. Now it belongs with the fossils. Unfortunately, as much as we would like to keep it running, our customers are the ones telling us to upgrade. We kept telling them each quarter that things would change, but if at the end of 2023 we still have absolutely no idea in terms of timeline, then this means that we are looking closer to 2030 than 2025. Sadly we will need to make a decision this month. Ah well, it was good while it lasted, I really liked it. But I also remember my old For Probe GT 1989, I liked that one too but one day it went to the graveyard. OST should not have waited so long to start with new UI, for many its already too late. Cheers all - M

**Reply by JediKev** [reply]
Admittedly, we haven't been very frequent with community updates for v2.0. However, this is simply because 1. it is still private (closed-source) as we are still working on core parts and 2. we are super focused on the rewrite (among other things)! With this being said, we hope to make more frequent community updates for v2.0 in the future. As it currently stands, we still do not have a set release date or anything so you'll just have to stay tuned for upcoming updates. I _can_ say that development is going really well and we are super excited to share it with the community as soon as possible. If you can't patiently wait for this _monolithic_ rewrite then that's your decision (just like other patiently restore and maintain their old car instead of throwing it in the graveyard 😉).

**Reply by makumba8888** [reply]
Thanks for the reply, much appreciated. It would be great if we could have a 'little peek' at what it will look like. I know we used that for our own software redesign with customers. Just 1 or 2 slides made a huge difference, even if it was a screenshot (not even coded). But between saying that its coming and showing 1 or 2 slides, it makes a big difference. Keep up the good work, we will see what we can do on our end.

**Reply by fre2mansur** [reply]
Why private repository?
Not anymore open-source?
Community can help to complete the road map, 

Why it's bit shady, not even telling what's going on? Rewriting? I thought you writing new code with laravel framework? You can publish about what passes are completed which phase we are at it? If you can share about small info, I think we are far away from the release, or you have big plan with 2.0, not only Development, but making it as a commercial.

Anyway. I don't see the transparent in 2.0 project.  I don't think you can answer those questions. Good luck.

**Reply by JediKev** [reply]
@fre2mansur 

We will make it open-source once we are ready. As I said above we are still working on core parts. We can’t let the public contribute yet as things are still changing and we are still working on core parts.

Full rewrite yes, because osTicket was traditionally hand written with no framework at all. Laravel will give us a framework to write less code and will allow more features, etc. that will be super helpful.

As I said above we still don’t have a timeline but we hope to have a preview/RC as soon as we can. If you cannot be patient then that’s on you :)

Cheers.

**Reply by fre2mansur** [reply]
Thank you for your hard work for both repo.

Just wondering Can't we keep the repo public while not accepting pullrequest?  Atleast we can test the part or just see the process, it will surely give us the confidence to wait for 2 more years.

**Reply by levik-opsilient** [reply]
Even without contributing code, if it's made public we can assist with qa.

**Reply by TVTS-Tyler** [reply]
Ive been waiting for v2 since it was first announced *checks watch* THREE years ago. And the only "updates" ever posted are saying "theres still no timeline". Even when people ask for a BALLPARK, the answer is still "no timeline". If after 3 years there's literally NOTHING of substance you can share with the community who is so eagerly awaiting this, it starts to make us wonder if this is indeed a project that will ever see the light of day, or if it's a pipe dream that will be stuck in development indefinitely, or until you all give up on it. If you're still working on core parts after this long, is it fair to assume no REAL progress has been made? Are we still working on fundamentals? Are we much past square one?

And your responses to people about not being able to be patient is their problem.. excuse me? after THREE YEARS I think we all have been plenty patient, and we deserve some answers. You seem to avoid actually answering any questions, but ill try anyway..

1. Do you expect, or even just hope, to have something the community can SEE in this calendar year? Even just a single screenshot? 
2. If No, is it because there is nothing too see? Is that how little progress has been made?

You admitted that youve been poor at sharing updates, so what can you share with us just past the 3rd year in development? ANYTHING other than "were working on core parts". We know, you say it a lot. What else though? Give us SOMETHING. People are holding out for this, but with no real idea of when we might see something, it's getting harder to justify. We aren't asking for a release date, or anything set in stone. Just a rough idea, your best guess at some progress. Even if it ends up being off, or taking a little longer. Just give us something....

**Reply by JediKev** [reply]
To answer your question yes something is coming this year; a preview.

**Reply by PPCM** [reply]
@TVTS-Tyler was a little bit rough, but this can be heard --> eagerness
And in the other hand, we have to be grateful with the team who dedicates their time to us.

@JediKev you made a great work with all features present in OST 1.18.1, but the UI is so old...
I have lot of difficulties to make my new users to accept to use OST, only for the look and feel 😒
Lot of IT teams are scared to receive the order to change OST by something else, only for a question of presentation

The community can give help, feel free to ask
For example, I just made dockers to deploy OST easily (hub[dot]docker[dot]com/r/ppcm/osticket-server)
If you need, it can be done for the 2.0 version, and why not for the preview. In that way, it will be easy to deploy, to test and to show to users
Feel free to contact me if the idea appeals to you

**Reply by GrahamSH-LLK** [reply]
>but the UI is so old

i made a userstyle for this, you can edit the domain in the text! https://userstyles.world/style/1865/osticket-redesign

**Reply by PPCM** [reply]
@GrahamSH-LLK seems to be a good contribution, do you have a preview somewhere?
And how to setup your CSS?

**Reply by JediKev** [reply]
@PPCM @GrahamSH-LLK please stay on topic. This is not a forum or chat site for third party mods.

**Reply by laserjet25** [reply]
> To answer your question yes something is coming this year; a preview.

@JediKev Trying to hold you to your word about the preview of v2. Do you have any expectation of when the community can see that any work has been done on v2? Some screenshots would really give me some hope that this project is still being worked on.

**Reply by JediKev** [reply]
@laserjet25 later this year

**Reply by laserjet25** [reply]
Thanks for getting back so quickly.

Are you able to elaborate why the OST team has been unable to communicate anything meaningful up to this point? I noted some of the concerns I feel people have in another reply here. I also know people in the forums have been asking for anything meaningful to show any works has been done and there has never been a response saying anything other than "it's being worked on but we can't show anything". I think anything will do at this point since the goal posts to show any progress has been made have been getting moved for years for seemingly no reason.

It has been extremely difficult to consider continuing with this project because of the lack of communication and updates since v2 theoretically materialized a few years ago.

**Reply by fre2mansur** [reply]
What about we fork new and merge your pull requests? And improve further?

**Reply by theking81** [reply]
> @laserjet25 later this year

Its getting later... lol

**Reply by waczny** [reply]
> > @laserjet25 later this year
> 
> Its getting later... lol

😃 2 more months… wish their project manager would at least throw some updates in here.

**Reply by laserjet25** [reply]
> @laserjet25 later this year

Holding you guys to this. Really looking forward to the possibility of OST being usable but I'm really getting the feeling of a highschooler procrastinating on getting his homework done... Not sure if I'll get a Christmas suprise or watch you guys get an F for missing your assignment date :)

Are you able to share anything on why there hasn't been any updates to this point? I don't really see the benefit of stringing the community along since that will likely (probably already has) kill your project here. Totally makes sense if you are planning on charging $$$$ to use this and kill the OSS version. Maybe management is living under a rock and doesn't check GitHub at all?

Project page still hasn't been updated since mid 2024.

**Reply by laserjet25** [reply]
@JediKev It doesn't really look good this is the second year in a row you have said this. As I and others have said, we can't really understand why the OSTicket team has gone completely radio silent on v2. All of this really suggests that there has been no meaningful progress. How do you expect your users or clients to hold confidence in your product or team with the way this has been handled? We have been strung along for a really long time now and you keep holding out on the idea that the product is alive but won't show anything.

It really makes no sense when you said you can't show anything unless it is perfect. Honestly you are doing more harm to the project with this horrible PR stunt.

Do you have any visible progress to show the community as you said?

> @laserjet25 later this year

> To answer your question yes something is coming this year; a preview.

**Reply by JediKev** [reply]
@laserjet25 Please read newer comments for the most up to date information. I’ve provided an update on this here:
- https://github.com/osTicket/roadmap/discussions/36#discussioncomment-15177761

**Reply by NotUrAverageITGuy** [reply]
@JediKev to piggyback off of @laserjet25. You haven't provided anything. Your latest comment says "Big News Coming and there will 100% be a beta release in 2026." Which how is that any different than what you have said before and how are we suppose to believe that when in April 2024 you said "Something is coming this year. A preview" and then in Feb 2025 you said "Later this year". I am sure I can speak for everyone that we are very excited to see the new product, just don't lie? Or if things have changed, be honest and say "we couldn't meet this date because ...." It's not hard to do.

Again as I have stated in the past, it is a free product, so you can't complain too much, but when one thing is said and then another is done, it's a very bad look.

Is the goal for the marketing plan to push away as many users as possible? How is just be patient still the go to unless we are just stringing people along? I understand it may not be ready, but what harm could you possibly do by giving a date when what you have done in the past is arguably worse? Are we talking Q1? Q3? Or will we be back again December 2026 asking the same questions? You commented you completely understand the frustration from the community, but there seems to be no learning from the decisions causing it?

I will be the first one in line praising it when or if it comes out, but it is 100% fair game to give the much deserved criticism.

**Comment by protich** [comment]
@TVTS-Tyler - EXCUSE you! No one works for you or owes you shit mate!

**Reply by laserjet25** [reply]
Hey there, long time lurker in the OST git/forum.

While TVTS-Tyler may have come across a bit strong, I think it sums up the general feeling of the community over the last few years. It's true this is an open source product and anyone is capable of contributing (in theory), but because the v2 project has been stuck since what seems at least 2021, v1.18 has stalled since 2023. I myself am not capable of adding any useful code to this project so I rely on others who have more skill in development. There are 2 features I have been hoping would make their way into one of the branches but to no avail:
- [Ticket Time Tracking](https://github.com/osTicket/osTicket/pull/3231)
- [Revolving SLA](https://github.com/osTicket/osTicket/pull/419)

The above pulls were started back in 2014, but never received comment by any maintainer. The thought process that anyone can contribute only works if the maintainers actively work with the community to add what they want. Open source software thrives on community involvement and the team behind this seem to have forgotten that. Every day we get another "we are working on v2 but have no up-to-date roadmap and the code is private" is another day members of this community feel strung along. I feel that I'm in the same situation as a lot of people here who are likely starting to consider alternatives because of this lack of communication. I'm glad that there is a roadmap, but that is only useful if updated. Most of the tasks are from 2022/23 and have had no change since creation.

You have failed to communicate as a CEO with your brash remarks and the OST team has failed to communicate a meaningful status of the v2 project. Try to understand why the community behind this project might be frustrated and respond appropriately. I as a user of this project am worried it has/will loose a portion of the userbase who want something that is being actively developed. That is never a good trend.

Open source projects are built for the community, by the community and die with the community.

**Comment by fre2mansur** [comment]
There is nothing to share. Don't expect anything.

**Reply by makumba8888** [reply]
yea, I was asking myself the same question a few weeks ago. Sadly back at the end of 2023 we decided to go with a commercial tool. Of course it works and of course our users and our customers are happy. But we are now tied with a commercial tool. It was that or keep getting told by our customers each two days that our customer support UI looked like it was made in the 70s. We'll see in the future but it would take a real kick-ass interface and new features to bring us back here because the commercial ones have the money to update all the time. For example, we are a SaaS business company and our UI gets updated often, if not we fall behind the pack real fast. Same here. Really wish the OST Team good luck if they can deliver a new UI but after what, 4-5 years of hearing about this it makes me think of aliens. We hear a lot about them but we never see them :-) I guess all of us here are at the point where we will believe it when we see it. But who knows. Only question I have is, if they started design 4-5  years ago, by the time they deploy, will the design already look old? Cycle times for business UIs these days are getting shorter each year. Good luck to all, I really liked the open source logic but today we find that our commercial tool has 10 times more features and built in reports, so we know our internal users and customers will probably never want to go back. - Cheers all - M

**Comment by mreimer-pixel** [comment]
Dear [Team/Developers],
I hope this message finds you well. I was wondering if there are any updates you could share regarding the project. I recall that the goal was to have a working prototype by the end of the year, and I fully understand that development takes time and that everyone has other commitments.
Please know that this is not a criticism in any way — I truly appreciate the effort you're putting into this project, especially since it's being done in your own time. That said, I believe the community would greatly benefit from occasional updates, even brief ones, to help us understand the current timeline or any challenges that might be causing delays.
Clear and consistent communication, even if it's just a short email or forum post every few months, would go a long way in keeping everyone informed and engaged.
Thank you again for your dedication and hard work. We’re all excited to see where this project goes!
Regards
MADMAX

**Reply by makumba8888** [reply]
I agree that updates would have been good. I think everyone on this board would have liked to have some kind of update in the past what, 4 years now. We used OST for a few years and internally, we liked it. But at this point there is another reality in play, one that the developers here have no control over: There are just way too many companies with lots of VC backed money that develop great products and that we can use for a reasonable price. And now, with the introduction of AI, some of these products are light years in advance of whatever will be developed here. There is simply no comparison. If you have a ticket system with full AI agents that can do 70 percent of the work you used to do and the agents, do it as good as you would, you will never want to go back to the <old> system. Here, they might come out with a new design, a new UI. But by the time they do that, the VC backed guys have a product that is 50 times better on the market. We used to use a lot of Open-Source products. We were super fans of open source. But in the past 3 years we see everything changing and now we see that it makes more sense to pay for tools that are way better, are updated all the time, provide us great support almost in real time. Yes, it is sad for those producing open source, working like crazy for free on their spare time. But the reality is that you get what you pay for. Even if OST gets a facelift, it will still be 10 years behind the other tools. Now, if your customers don't care, that might be ok. But customers now are more and more aware of what is out there. They get service from other companies so they can compare between different customer support. We actually came to that exact point where our customers were the ones telling us to ditch OST and get into the 2000s. We listened and we dramatically increased the quality of our support system. Our customers were also happier, and they told us point blank.
So, of course each company does what it wants. But if you are holding out, waiting for a miracle with a new version here, it is not going to happen. Today it is impossible to compete with companies that are fully funded, unless you are in a field that is completely under the radar. And a ticket support software is not one of those. Plenty are commercially available.

**Reply by TVTS-Tyler** [reply]
Until today I assumed the project was dead. After the snails pace at which anything seemed to be happening, I gave up hope and ditched plans to use it if/when it ever came out.

But it appears that finally some real information has been trickling through. I see there is a dedicated PM, which was probably long overdue, and that's exciting. The main reason I initially looked for a v2 was for a better experience on mobile devices, so it's reassuring to see that it should be a fully responsive UI.

Ultimately, Im glad to see this project finally start gaining some good forward momentum, even if I think updates of some kind, even just better communication, was due a long time ago. If for nothing else, the sake of the IT people who have been trying desperately to convince bosses/management to stay on v1 and hold out for v2 instead of having to leave the platform entirely. People were dying for ANY scrap of info they could show as justification to keep waiting for v2. In a project like this I do personally feel like the team owed the community something in that regard. Anyhow, the idea here is not to rehash old squabbles. 

So, will I use v2 when it comes out? I'll definitely play with it and see what its like. But actually putting it into production will ultimately depend on the timeline and how long it takes to have a stable release out. I'm dying for a preview that, last I saw, was on track to come out this year! Fingers crossed. To the devs, sorry for my rant last time. Perhaps I was a little aggressive and should have made my points in a more constructive way. I felt I spoke for a lot of people here with my frustration, but it was unproductive to approach it the way I did. Looking forward to more good news and hopefully a sneak peak before the new year.

**Reply by JediKev** [reply]
Trust me, I completely understand where people are coming from. It pains me to see someone frustrated because we care so deeply about our products and the people who adopt them. We are extremely proud that we hold a legacy spanning over 20 years. So I hope everyone can understand that we do not want to release a half-baked or rudimentary product/demo. Our rewrite (v2.0) is so important and so deep for us that everything about it **_HAS_** to be nothing short of amazing. About a year or so ago we further expanded our team with some amazing engineers. Between then and now we have accomplished an insane amount of progress and added much more than we ever dreamed. Big things are coming soon so hang tight, we’re almost ready to show you what’s next. I can’t say too much but what I can say is the UI/UX is super responsive, visually gorgeous, and most importantly it doesn’t look like it’s from the 90's!! Oh and no, we are **_not_** going to charge money for v2.0. It will remain free and open source; and that’s crazy in the big 2025.

**Reply by TetraAsh** [reply]
> Trust me, I completely understand where people are coming from. It pains me to see someone frustrated because we care so deeply about our products and the people who adopt them. We are extremely proud that we hold a legacy spanning over 20 years. So I hope everyone can understand that we do not want to release a half-baked or rudimentary product/demo. Our rewrite (v2.0) is so important and so deep for us that everything about it **_HAS_** to be nothing short of amazing. About a year or so ago we further expanded our team with some amazing engineers. Between then and now we have accomplished an insane amount of progress and added much more than we ever dreamed. Big things are coming soon so hang tight, we’re almost ready to show you what’s next. I can’t say too much but what I can say is the UI/UX is super responsive, visually gorgeous, and most importantly it doesn’t look like it’s from the 90's!! Oh and no, we are **_not_** going to charge money for v2.0. It will remain free and open source; and that’s crazy in the big 2025.



Thats great news, i came checking as i was starting to think the project abandoned. 

Any sneak previews you can give us e.g. what were the 5 biggest aims of the re-write e.g. the new UI

Any news on if the new system will have a better API? Its the big thing i thought was missing/poorly executed in the current version!

**Reply by fre2mansur** [reply]
> > Trust me, I completely understand where people are coming from. It pains me to see someone frustrated because we care so deeply about our products and the people who adopt them. We are extremely proud that we hold a legacy spanning over 20 years. So I hope everyone can understand that we do not want to release a half-baked or rudimentary product/demo. Our rewrite (v2.0) is so important and so deep for us that everything about it **_HAS_** to be nothing short of amazing. About a year or so ago we further expanded our team with some amazing engineers. Between then and now we have accomplished an insane amount of progress and added much more than we ever dreamed. Big things are coming soon so hang tight, we’re almost ready to show you what’s next. I can’t say too much but what I can say is the UI/UX is super responsive, visually gorgeous, and most importantly it doesn’t look like it’s from the 90's!! Oh and no, we are **_not_** going to charge money for v2.0. It will remain free and open source; and that’s crazy in the big 2025.
> 
> Thats great news, i came checking as i was starting to think the project abandoned.
> 
> Any sneak previews you can give us e.g. what were the 5 biggest aims of the re-write e.g. the new UI
> 
> Any news on if the new system will have a better API? Its the big thing i thought was missing/poorly executed in the current version!

Don't even bother, you will get nothing, we have been hearing this lie for a decade.

**Reply by JediKev** [reply]
@fre2mansur Hey Negative Nancy! Nice to see you again. I think you're our biggest fan! You comment on everything; like an ex who truly can't let things go! 🤣 But don't worry, we are not the ones to hold grudges. We will still welcome you with open arms once v2.0 is released. 👐🏻 ❤️ 

@TetraAsh Sure thing! I can't say too much but UI/UX complete revamp/overhaul/modernization, focus on better WCAG support, fully RESTful API, all core functionality but improved, new features, etc. It's going to be a MASSIVE update and will make all the waiting worth it!

**Reply by KratosGemini** [reply]
@JediKev, thanks a bunch for the teaser update! It's nice to simply hear that progress is still being made.

We have an OS Ticket instance, and I'm considering a ticket system to help with a new project....deciding if it makes sense to go another direction for it or try to use OS Ticket as well and see if expanding our usage makes sense.

**Reply by TetraAsh** [reply]
> We have an OS Ticket instance, and I'm considering a ticket system to help with a new project....deciding if it makes sense to go another direction for it or try to use OS Ticket as well and see if expanding our usage makes sense.

That's really going to be a matter of figuring out project needs, with no fixed timescale on the next version, you shouldn't put any reliance on that, so look at OSTicket as it currently stands and decide if it fits your needs. 


@JediKev 
Likely pointless to raise at this stage but two key elements from a use case side. 
1) We had to completely build a new search page because the built in search is horrendous and rarely finds what's needed. The page i built is a simple Search direct in the SQL DB and works fine. Please fix the search features to find tickets based on keywords and dont over think it, simple is better. 
2) We built another page, as a helpdesk we get calls from customers, what we did was set up a page that was called from our phoen software to www.ticketsystem.com/usersearch?q=01234567890 (the phone number calling in), this page then pulls the users account, shows some core info at the top and a list of the users open tickets, and the companies open tickets. 
This is core starting info for our engineers to know who's calling and start support for them quickly. A similar page built in would be amazing for ourselves and others that run a call in helpdesk.

**Reply by NotUrAverageITGuy** [reply]
> Trust me, I completely understand where people are coming from. It pains me to see someone frustrated because we care so deeply about our products and the people who adopt them. We are extremely proud that we hold a legacy spanning over 20 years. So I hope everyone can understand that we do not want to release a half-baked or rudimentary product/demo. Our rewrite (v2.0) is so important and so deep for us that everything about it **_HAS_** to be nothing short of amazing. About a year or so ago we further expanded our team with some amazing engineers. Between then and now we have accomplished an insane amount of progress and added much more than we ever dreamed. Big things are coming soon so hang tight, we’re almost ready to show you what’s next. I can’t say too much but what I can say is the UI/UX is super responsive, visually gorgeous, and most importantly it doesn’t look like it’s from the 90's!! Oh and no, we are **_not_** going to charge money for v2.0. It will remain free and open source; and that’s crazy in the big 2025.

@JediKev I am genuinely happy to hear that we are close to a release, but no disrespect there are threads that say a snippet was coming by end of 2024, so saying "big things are coming soon" makes people hesitant. I've been a dedicated user for about 6 years now and have no plans on moving because v1 works well for what it is and I don't see a need to pay for my orgs use case.

Two questions if you may.  What is the reason for the secrecy on timeline or updates in general? Obviously if you committed that something was to come say Dec 24 that would be a bad look considering it's Dec 25. Is it just personal preference to not give any updates, whether it be like  screenshots of new features/UI or just keeping the 2.0 GitHub page updated? Or was there not anything ready? Or did the project get completely revamped and what's you said in 2024 was no longer relevant? I like many other disagree with how the marketing for v2 has gone, but it is completely reasonable that as a free product, you don't owe your user base anything, because they do not contribute financially. However I wonder if you agree that communication practices need to be taken a look at because saying something is coming, but then not delivering is not a good look.

Final question, in the spirit of Christmas, is what's being realized this year (if still the case) a fully usable product with migration capabilities from current v1 or just visual, here is what we have so far?

Merry Christmas!

**Reply by mlipok** [reply]
> Final question, in the spirit of Christmas, is what's being realized this year (if still the case) a fully usable product with migration capabilities from current v1 or just visual, here is what we have so far?

It's quite possible that this still works with the same database structure, or only slightly modified, or perhaps without modifying the existing data in the database tables, but only adding new data.

This is interesting.
Could the team clarify this?

**Reply by JediKev** [reply]
I’ve touched on this before but it’ll be a full upgrade path just like current legacy.

**Reply by thompol** [reply]
@JediKev I think what would help is a announcement here, on the website, etc. But this must feature some exact dates, so it depends on when you're ready to share that ;)

**Reply by javiervalerocelada-creator** [reply]
@JediKev Are you planning to release a "rc" this year?

**Comment by stevland** [comment]
For all the naysayers:

https://next.osticket.com/

**Reply by thompol** [reply]
Never doubted you, even though our patience was tested :) Amazing, looking forward to what's coming.

**Reply by stevland** [reply]
> Never doubted you

To be 100% clear, I'm not a part of the team. Just an osTicket devotee who is looking forward to v2 as well. ;)

**Reply by JediKev** [reply]
We are going to post something on GitHub soon. @stevland just beat us to it 😂

**Reply by stevland** [reply]
> We are going to post something on GitHub soon. @stevland just beat us to it 😂

Oops.

Sorry to steal your thunder! 😂

**Reply by TetraAsh** [reply]
Amazing news. Well done to the whole team working on it, it really looks like a great concept. 
I'm now just itching to get my eyes on the code to start our own development and integration process. 
I do fear that we may still be 6-12 months off a RC1 release though! 🤣

But at least i have time to learn Laveral....🤣

**Reply by mlipok** [reply]
If we get RC1 then I be able to run copy of current setup and check all new things.
This will be a very important and beneficial time for me. That's why I'm actually glad it will be at least six months before the official release.

**Comment by JediKev** [comment, marked as answer]
I’m beyond ecstatic that I finally get to close this with a real, tangible update. Please read the osTicket 2.0 Announcement (#72) and [subscribe to the mailing list](https://next.osticket.com/?ssd=true) to stay updated! Yes, it’s really happening folks!
